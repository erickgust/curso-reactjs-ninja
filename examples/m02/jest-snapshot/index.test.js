'use strict'

const responseFromServer = () => [
  'react',
  'javascript',
  'node'
]

it('Testing snapshots', () => {
  expect(responseFromServer()).toMatchSnapshot()
})
