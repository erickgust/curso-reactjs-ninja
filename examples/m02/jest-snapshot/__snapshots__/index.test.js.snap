// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing snapshots 1`] = `
Array [
  "react",
  "javascript",
  "node",
]
`;
